RA-193: Update SLA (#171)
equinor/radix-platform: public-site/site/docs/topic-sla/index.md

---

# Service level agreement

We are experimenting with an approach with "release channels" which differ in update frequency, service level agreement(SLA) and expected stability. See [link](/public-site/site/guides/getting-started/index.md#the-radix-clusters) for more information on clusters

|    Cluster            |             Purpose                              |      Termination/Upgrade    |   Support    |     
|-----------------------|--------------------------------------------------|:---------------------------:|:------------:|

> | **Production**        | Radix production - currenly limited access       | Every ~6 months             | Yes          |   

Reviewer: Fjerne "limited access"

| **Playground**        | "Open" for test and experimenting with Radix     |                             | Best-effort  |  

## SLA - Production 

> Should be used when your team has chosen radix as PaaS for a product under development or in production. 

Reviewer: Radix as Paas (Platform as a Service)

Radix stor R, og forklaring på akronym

### Uptime

  * **Platform monthly uptime: 99.5%** - expected uptime for radix as a hosting platform. 
  * **Omnia Radix services monthly uptime: 98%** - expected uptime for radix services, as CI/CD and monitoring. 
  * **Planned maintenance:** We will announce planned maintenance at least 2 business days in advance. Downtime during planned maintenance does not count towards uptime goals.
  
### Associated operational risks
- No incident management beyond schedule "Norway - default" - i.e. no support after 16:00 CET/CEST on Norwegian working days.
- Risks for infrastructure downtime despite robust, high availability infrastructure.
- Disaster recovery in experimentation for platform and possibly apps.

## SLA - Playground

Use Playground for testing Radix, see if it’s a good fit for your projects, and provide feedback to us. When you are ready to commit, you can register your application in the Production cluster, which has improved stability.

**Support:** same channel as for Production cluster. Help will be provided when team has the time. 

**Uptime:** "Best-effort", but no guarantee uptime. Planned maintenance is announced as early as possible. 

> **In Playground cluster - hosted applications might be lost during maintenance, upgrades or migrations.**

Reviewer: ```suggestion
**In Playground cluster - hosted applications **may** need to be re-registered after maintenance, upgrades or migrations.**
```
